src/components/footer.tsx

```diff
@@ -72,65 +72,45 @@ const AutoScrollLink: React.FC<any> = props => {
   return <Link onClick={scrollToContent} {...props} />
 }
 
-const classifiedAdminAreas = config.ADMIN_AREAS.map(area => ({
-  areaClass: `${regionId}-${area.toLowerCase()}`,
+const classifiedAdminAreas = config.ADMIN_AREAS.map(area => [
+  `${regionId}-${area.toLowerCase()}`,
   area
-}))
+])
 
 export const Footer: React.FC = props => {
   const { t, i18n } = useTranslation()
 
+  let aboutMenu: any = i18n.t('footer.aboutMenu', { returnObjects: true })
+  if (!(aboutMenu instanceof Array)) aboutMenu = []
```

Comment:
There's also a method on i18next to check existence of a key IIRC

Reply:
🙌 `i18next.exists(key, options)` THANK YOU